x-pack/plugins/security_solution/scripts/endpoint/common/screen/column_layout_formatter.ts

```diff
@@ -84,3 +83,15 @@ export class ColumnLayoutFormatter extends DataFormatter {
     return colData + (fillCount > 0 ? ' '.repeat(fillCount) : '');
   }
 }
+
+// this is a copy of the `ansiRegex` module, which is no longer allowed to be `import`ed at the module
+// because it is not defined as an ESM module (only dynamic import is supported). Due to its usage
+// which does not allow `async` calls to be `await`ed, it is not being copied here for use locally.
```

Comment:
I guess there's an unnecessary `not` in the explanation in the second half of line 89

Reply:
Thanks @gergoabraham . I corrected it.